pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableappprofile/_http.log

```diff
@@ -363,21 +363,7 @@ X-Xss-Protection: 0
   "metadata": {
```

Comment:
This comment is for `"done": true,` on line 361. From https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4398/commits/32ee880971f74f492edd98e2ec299c2a6137fcc7 it seems that done should be false? Or did I miss anything from the real log?

Reply:
I'm not sure I understand the question. I believe the real GCP response doesn't include `"done":`, but based on the observed behavior (there is no follow-up GET request on the operation), I think the operation is "done".

The `"done": true` field is something our mock introduces. I'm not sure how to remove it, but I think it's safe to ignore since the behavior is consistent between the mock and the real service.